btr-web/btr-main-app/cypress/e2e/pages/requestOmit.cy.ts

```diff
@@ -0,0 +1,59 @@
+describe('pages -> Request To Omit', () => {
+  let i18nCommon: any
+  let i18n: any
+  beforeEach(() => {
+    cy.readFile('../btr-common-components/lang/en.json').then((json) => { i18nCommon = json })
```

Comment:
I think you dont need necesarily to `readFile` I think just importing it as constant would work too.

Reply:
I was going this for consistency with the formValidation tests